client/src/components/molecules/AuthForm/index.tsx

```diff
@@ -40,7 +44,7 @@ const AuthForm: FC<Props> = (props): JSX.Element => {
           <input
             type="text"
             css={globals.form_control}
-            disabled={isSubmitting}
+            disabled={isLoading}
```

Comment:
Have you tried sir about like this

`disabled={isSubmitting | isLoading}`

if you are comfortable for that implementation please disregard this message. Thankyou.